server/routes/shared_api/autocomplete/helpers.py

```diff
@@ -22,11 +22,41 @@
 import requests
 
 from server.routes.shared_api.autocomplete.types import ScoredPrediction
+from server.routes.shared_api.place import findplacedcid
 
 MAPS_API_URL = "https://maps.googleapis.com/maps/api/place/autocomplete/json?"
 MIN_CHARACTERS_PER_QUERY = 3
 MAX_NUM_OF_QUERIES = 4
 DISPLAYED_RESPONSE_COUNT_LIMIT = 5
+TWO_WORD_CONTINENTS = [{
```

Comment:
I could see us wanting to match other places at some point that aren't accounted for by the Google Maps API. Maybe instead of calling this `CONTINENTS`, it could be something like `ADDITIONAL_PLACES` ?

Reply:
So I specifically named it continents because I think we shouldn't be doing this for anything else. This is just a stopgap because we can't launch place autocomplete without continents. But I think we should use whatever we will for StatVars for everything custom in the future.
I can still do the renaming, but this is pretty hacky, so I do want to emphasize we shouldnt make this a precedent for throwing in other things :)